scrnaseq.Rmd

```diff
@@ -556,24 +911,25 @@ Take care not to mis-read a UMAP:
   
 For a nice read to intuitively understand UMAP, see `r knitcitations::citep("https://pair-code.github.io/understanding-umap/")`. 
 ```{r umap, message=FALSE, results="hide"}
+# Default UMAP
 sc = Seurat::RunUMAP(sc, dims=1:param$pc_n)
 
+# 3D UMAP
+sc = Seurat::RunUMAP(sc, dims=1:param$pc_n, n.components=3, reduction.name="umap3d", reduction.key="UMAP3D_")
```

Comment:
Cool idea.

Reply:
Are we plotting it anywhere? Don't see it yet.

Reply:
No, it only makes sense when we export it to Cerebro. There you can zoom and rotate.

Reply:
I see.